index.js

```diff
@@ -1,18 +1,63 @@
 'use strict';
 
-module.exports = function implicitFiguresPlugin(md) {
+var figure = function (options) {
 
-  var defaultRender = md.renderer.rules.image;
+  var dataType = (typeof options.dataType !== 'undefined') ? options.dataType : false;
 
-  md.renderer.rules.image = function (tokens, idx, options, env, self) {
-    var token = tokens[idx];
-    var img = defaultRender(tokens, idx, options, env, self);
+  return function implicit_figure (state, startLine, endLine, silent) {
```

Comment:
Good question! Comes straight from the `markdown-it` core paragraph handler. I'm sure some trimming can be done.